PhantomConnectExample/PhantomConnectExample/ContentView.swift

```diff
@@ -8,17 +8,25 @@
 import SwiftUI
 import Solana
 import PhantomConnect
+import Foundation
 
 struct ContentView: View {
     
     @StateObject var viewModel = PhantomConnectViewModel()
     
-    @State var walletConnected = false
+    /// Saving keys in appstorage(user defaults).
+    /// THIS IS NOT SECURE! FOR DEMO ONLY!
+    /// Keychain can be used as a ssecure storage
+    @AppStorage("k_wallet_connected") var walletConnected = false
+    @AppStorage("k_wallet_public_key") var walletPublicKeyB58: String?
+    @AppStorage("k_wallet_phantom_encryption_key") var phantomEncryptionKeyB58: String?
     @State var walletPublicKey: PublicKey?
     @State var phantomEncryptionKey: PublicKey?
-    @State var session: String?
+    @AppStorage("k_wallet_phantom_session") var session: String?
     @State var transactionSignature: String?
     
+    @State var balance_Sol: Double?
```

Comment:
Let's keep formatting similar here and use camel casing rather than snake casing.